frontend/src/employee-frontend/components/document-templates/template-editor/DocumentTemplatesPage.tsx

```diff
@@ -139,20 +156,31 @@ const TemplateRow = React.memo(function TemplateRow({
             disabled={template.published}
             onClick={() => deleteDocumentTemplate(template.id)}
           />
+          <IconButton
```

Comment:
I would prefer presenting the download as a link with `target="_blank"`. This would allow the user to use context menu to "Save target as".